### ISSUE TITLE
Update queue types

### ISSUE THREAD

**Issue by fculpo** [meraki-analytics/cassiopeia#85]
Hi,

I tried to use Cassiopeia yesterday, but some queues are not defined, like TEAM_BUILDER_RANKED_SOLO.
See the output when trying to run the example:
```
Traceback (most recent call last):
  Match ID: 2373942625
  File "E:/Workspace/cassiopeia/examples/parse_match_information.py", line 87, in <module>
  Version/Patch: 6.24.169.4424
  Creation date: 2016-12-16 05:06:10.941000  (which was 12:58:38.125997 ago)
  Duration: 0:22:24
    main()
  File "E:/Workspace/cassiopeia/examples/parse_match_information.py", line 31, in main
    print("  Queue: {0}".format(match.queue))
  File "E:\Workspace\cassiopeia\cassiopeia\type\core\match.py", line 121, in queue
  Map: Map.summoners_rift
    return cassiopeia.type.core.common.Queue(self.data.queueType) if self.data.queueType else None
  File "c:\python35\Lib\enum.py", line 241, in __call__
  Mode: GameMode.classic
    return cls.__new__(cls, value)
  Type: GameType.matched
  File "c:\python35\Lib\enum.py", line 476, in __new__
  Platform: Platform.north_america
    raise ValueError("%r is not a valid %s" % (value, cls.__name__))
ValueError: 'TEAM_BUILDER_RANKED_SOLO' is not a valid Queue
```

Could you update the constants like the ones in type/core/common.py to reflect last API changes ?

Thx

**Comment by jjmaldonis** [comment]
Thanks for the report. If you can submit a PR for this it will speed things up for us. Otherwise we'll try to get to it tonight or in the next few days. You can see a similar PR here: #82 #84

**Comment by fculpo** [comment]
submitted #87

**Comment by fculpo** [comment]
Merged #87
